mmr-backend/api.py

```diff
@@ -11,19 +12,8 @@
 current_dir = os.path.dirname(__file__)
 sample_data_path = "test/sample-data/books.json"
```

Comment:
"For testing" code debería ir al directorio de test. No creo que haya ninguna razón funcional para que esto siga aquí, ahora que estás refactorizando.